azurerm/internal/services/apimanagement/client/client.go

```diff
@@ -7,6 +7,7 @@ import (
 
 type Client struct {
 	ApiClient                  *apimanagement.APIClient
+	ApiDiagnosticClient        *apimanagement.APIDiagnosticClient
```

Comment:
this client needs to be initialized, at the moment running the tests causes a panic - we can do that below: https://github.com/terraform-providers/terraform-provider-azurerm/pull/7873/files#diff-e334dbeb932a241f1dd03111cd3f4d71R42-R43

Once that's done we should be able to re-run the tests here

Reply:
Should be done now in e521a8be0.